modules/serverless-service-mesh-resources.adoc

```diff
@@ -122,14 +106,11 @@ spec:
     kind: Service
     name: istio-ingressgateway
 ----
-
 <1> The {product-title} route must be created in the same namespace as the
-ServiceMeshControlPlane. In this example, the ServiceMeshControlPlane is
+`ServiceMeshControlPlane`. In this example, the `ServiceMeshControlPlane` is
```

Comment:
Do you need call-outs to describe what `host`, `port`, and `to` are?

Reply:
@markusthoemmes @cardil @nak3 thoughts?

Reply:
I think maybe since these docs already existed prior to this PR and it's sort of out of the scope because we're not modifying this YAML with this PR, let's add a Jira instead to document Routes properly and update it then.

Reply:
Added a larger issue to improve Route, Revision etc docs for Serving. We can follow up explaining all the fields for Routes there: https://issues.redhat.com/browse/SRVKS-783